### PR TITLE
3rd party dev faq updates

### PR DESCRIPTION
**Pull request by jospoortvliet**
Some updates to the FAQ about 3rd party app development. Sorry for the whitespace and formatting mess... I don't know what I was thinking.

### CODE REVIEW

**Comment by jospoortvliet** [comment]
@phil-davis your input is always welcome!

File: page-faq.php [hunk 1]
```diff
@@ -38,13 +38,14 @@
 	<li><a href="#howitstarted">How did ownCloud get started?</a></li>
 	<li><a href="#getinvolved">How can I get involved in the ownCloud Community?</a></li>
 	<li><a href="#notechknowledge">But what if I'm not very technical?</a></li>
+	<li><a href="#mobile">How can I get started writing mobile (andoid, iOS) clients or third party (web) apps?</a></li>
```
Comment (phil-davis) [inline page-faq.php]: s/andoid/android/

File: page-faq.php [hunk 2]
```diff
@@ -228,26 +243,39 @@
...
 </ul>
+
 <a name="notechknowledge"></a>
 <h3>But what if I'm not very technical?</h3>
 <ul>
 	<li>There are lots of ways you can contribute to ownCloud. We have a need for developers, designers, event organizers, speakers, QA and more. It is all <a href="/contribute/" target="_blank">on the <em>contribute</em> page</a>!</li>
 </ul>
+
+<a name="mobile"></a>
+<h3>How can I get started writing mobile (andoid, iOS) clients or third party (web) apps?</h3>
```
Comment (phil-davis) [inline page-faq.php]: More "andoid"

File: page-faq.php [hunk 3]
```diff
@@ -228,26 +243,39 @@
...
+<ul>
+	<li>First of all, awesome you want to bring ownCloud support to your app or built an app entirely for ownCloud, rocking!</li>
```
Comment (phil-davis) [inline page-faq.php]: s/built/build/

File: page-faq.php [hunk 4]
```diff
@@ -228,26 +243,39 @@
...
+	<li>If you're interested in writing and Android or iOS app with ownCloud support, you can use our MIT licensed libraries on those platforms. Find them on Github: <a href="https://github.com/owncloud/ios-library">iOS</a>, <a href="https://github.com/owncloud/android-library">Android</a>. You can then use the <a href="<?php echo $DOCUMENTATION_DEVELOPER; ?>ios_library">iOS</a> and <a href="<?php echo $DOCUMENTATION_DEVELOPER; ?>android_library">Android</a> developer docs.</li>
```
Comment (phil-davis) [inline page-faq.php]: "and Android or" => "an Android or"

**Comment by cparnot** [comment]
Adding the question about iOS/Android clarifies things a lot, thanks.

I noticed a couple of typos in the answer:

- "or built an app entirely" --> "or build an app entirely"
- "writing and Android or iOS app" --> "writing an Android or iOS app"

File: page-faq.php [hunk 5]
```diff
@@ -278,25 +310,31 @@
 </ul>
 
 <a name="closedapps"></a>
-<h3>Can I write closed source or proprietary apps for ownCloud?</h3>
+<h3>Can I write closed source or proprietary apps for ownCloud Server?</h3>
```
Comment (schiesbn) [inline page-faq.php]: Now that you also explain how to write external apps it doesn't make sense to add the "Server" part here.

Comment (jospoortvliet) [inline page-faq.php]: I rather not have people read that item at all if they want to develop an external app, but just go right away to the one talking about 3rd party apps. So I'll keep it in ;-)

**Comment by jospoortvliet** [comment]
Thanks everybody for the feedback. Implemented!

**Comment by jospoortvliet** [comment]
@cparnot and awesome you pinged back here. Good luck with your app and thanks for your contribution!

**Pull request merged**